script/Config/App.lua

```diff
@@ -96,11 +111,14 @@ Config.gen = {
 
 Config.game = {
   gameMode = 0, -- used by LTheoryRedux: 0 = undefined (splash screen), 1 = Startup Mode (Main Menu), 2 = Flight Mode
-  flightModeActive = false, -- flag for being in Flight Mode but unable to fly (as when player ship is destroyed)
+  flightModeButInactive = false, -- flag for being in Flight Mode but unable to fly (as when player ship is destroyed)
```

Comment:
Would this be better named as flightModeLocked ?
Consider whether this should actually be another game mode "3 = Flight Mode (locked)" rather than having an extra boolean around?